daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/infoset/Infoset.scala

```diff
@@ -295,6 +304,9 @@ class JDOMInfosetOutputter()
 
   override val infosetOutputter = new SJDOMInfosetOutputter()
 
+  /**
+   * Get the jdom2 Document representing the infoset created during a parse
```

Comment:
This comment isn't 100% the same as the comment that was added to the japi file.  The other comment said "Get the jdom Document" instead of "Get the jdom2 Document".  The two sets of APIs do have to be different where it's needed but it's a maintenance burden to have to make sure unwanted differences don't cause problems and this is an example of how easily differences can happen.